owlbot.py

```diff
@@ -19,6 +19,25 @@
 from synthtool import gcp
 from synthtool.languages import python
 
+default_version = "v2"
```

Comment:
I'm pretty sure this was added back by accident. The `bigquery_v2` modules are legacy and only for some backwards compatibility use cases.

Reply:
Originally removed in in https://github.com/googleapis/python-bigquery/commit/76d88fbb1316317a61fa1a63c101bc6f42f23af8